.nvmrc

```diff
@@ -0,0 +1,2 @@
+16.14.0
```

Comment:
Could you pin this to the major version 16 so nvm won't download this specific version if there is another node 16.x already installed? Also helps in case there is a newer 16.x version installed on the system, that will be used instead.

Reply:
Definitely!